beacon-chain/core/validators/validator.go

```diff
@@ -75,7 +75,7 @@ func InitiateValidatorExit(ctx context.Context, s state.BeaconState, idx primiti
 
 	// Compute exit queue epoch.
 	if s.Version() < version.Electra {
-		// Relevant spec code from deneb:
+		// Relevant spec code from phase0:
```

Comment:
This is part of why fork logic branching is very confusing. Should we refer to the latest version of the spec prior to electra or the version in which the logic was added? The thinking of the original comment was "this is the relevant spec code prior to electra" i.e. the spec code as it was in deneb. 

I don't feel strongly about this comment/change because it shouldn't exist with proper package scoping.

Reply:
I still fixed it because leaving it saying `deneb` is clearly a mistake